system/handlers/renderers/content/BooleanBadge.cfc

```diff
@@ -0,0 +1,35 @@
+component {
+
+	property name="presideObjectService" inject="presideObjectService";
+
+	public boolean function default( event, rc, prc, args={} ) {
+		return IsTrue( args.data ?: "" );
+	}
+
+	public string function admin( event, rc, prc, args={} ) {
+	
+		var data           = args.data         ?: "";
+		var objectName     = args.objectName   ?: "";
+		var propertyName   = args.propertyName ?: "";
+		var valueAttribute = "booleanBadgeUnknownValue";
+		var styleAttribute = "booleanBadgeUnknownStyle";
+		
+		if ( IsBoolean( data ) ) {
+			valueAttribute = data ? "booleanBadgeTrueValue" : "booleanBadgeFalseValue";
+			styleAttribute = data ? "booleanBadgeTrueStyle" : "booleanBadgeFalseStyle";
+		}
+		
+		var booleanBadgeValue = presideObjectService.getObjectPropertyAttribute( objectName=objectName, propertyName=propertyName, attributeName=valueAttribute, defaultValue="" );
+		var booleanBadgeStyle = presideObjectService.getObjectPropertyAttribute( objectName=objectName, propertyName=propertyName, attributeName=styleAttribute, defaultValue="info" );
+		
+		if ( isEmpty( booleanBadgeValue ) ) {
+			return "";
+		}
+		
+		booleanBadgeValue = translateResource( uri="preside-objects.#objectName#:field.#propertyName#.#valueAttribute#", defaultValue=booleanBadgeValue );
+		booleanBadgeValue = translateResource( uri="preside-objects.#objectName#:field.#propertyName#.#styleAttribute#", defaultValue=booleanBadgeStyle );
```

Comment:
Empty check need to move down below the i18n logic...
```suggestion
		booleanBadgeValue = translateResource( uri="preside-objects.#objectName#:field.#propertyName#.#valueAttribute#", defaultValue=booleanBadgeValue );
		booleanBadgeValue = translateResource( uri="preside-objects.#objectName#:field.#propertyName#.#styleAttribute#", defaultValue=booleanBadgeStyle );
		
		if ( isEmpty( booleanBadgeValue ) ) {
			return "";
		}
```

Reply:
fixed